CHANGELOG.rst

```diff
@@ -14,6 +14,18 @@ Change Log
 Unreleased
 --------------------
 
+[3.2.13] - 2020-05-19
+---------------------
+
+* Converting UUID fields to string for use in can_use_enterprise_catalog
```

Comment:
should these be combined? they have the same release version, and neither matches what's below...

Reply:
@brittneyexline Spoke with @chavesj about this. I believe he just needs to rebase this branch with `master`. Most of the changes in this PR were actually merged from #773, but uses the same branch of `chavesj/ENT-2885`. After rebasing, this PR could be significantly smaller.

Reply:
Disregard me, I saw you mentioned rebasing in the overall review comment :)

Reply:
i didn't realize that entirely but it makes sense. i felt like i had pr review dejavu...